### DISCUSSION TITLE
`dep ssh` host-labels

### DISCUSSION THREAD

**Discussion by staabm** [deployphp/deployer#2373] [General]
we have a config like

```
// ...
host('18.192.24Y.XXX', '18.158.22Y.XXX')
    ->set('deploy_path', '/my/path')
    ->user('my-user')
    ->stage('test');
// ...
```

this leads to a listing when invoked with `dep ssh`.

![grafik](https://user-images.githubusercontent.com/120441/104430000-7b725080-5586-11eb-8928-5a734c63783b.png)

is it possible to have 'labels' or something instead of just IPs to give the user a hint what a IP is refering to (the 2nd and 3rd entry in the list)?
with the display as can be seen on the screenshot, a user needs to know the IPs by heart which is not that useful.

**Comment by antonmedv** [comment]
Yes, it's possible. In Deployer each host has two properties:
 * hostname (real_hostname in v6)
 * alias (hostname in v6)

You can give names to hosts via aliases and set the correct IP. Here is an example for v7:
```php
set('deploy_user', '/path'); // In v7 it's possible to set deploy_path as recipe config.
set('remote_user', 'my-user');

host('foo')
    ->setHostname('18.192.24.01')
    ->setLabels(['stage' => 'test']);

host('bar')
    ->setHostname('18.192.24.02')
    ->setLabels(['stage' => 'test']);
```

**Reply by staabm** [reply]
thx, this works for me. <3
